common/models/user.js

```diff
@@ -1,3 +1,33 @@
+var mailerService = require('../../server/server-utils/emailProvider');
+
 module.exports = function(user) {
+    var User = user;
+
+
+    User.on('resetPasswordRequest', function(info){
+      var MailService = new mailerService();
```

Comment:
Done. Changed mailerservice to simply export a function that accepts a mail object and a callback.